python_modules/dagster/dagster/_core/definitions/assets.py

```diff
@@ -885,7 +885,13 @@ def subset_for(
                 freshness_policies_by_key=self.freshness_policies_by_key,
             )
 
+    @public
     def to_source_assets(self) -> Sequence[SourceAsset]:
+        """Returns a SourceAsset for each asset in this definition.
+
+        Each SourceAsset will have the same key, metadata, io_manager_key, etc. as one of the assets
```

Comment:
nit: I think the following is clearer:
```suggestion
        Each produced SourceAsset will have the same key, metadata, io_manager_key, etc. as the corresponding asset
```